physx-sys/src/lib.rs

```diff
@@ -386,6 +392,14 @@ extern "C" {
 
     pub fn get_alloc_callback_user_data(alloc_callback: *mut PxAllocatorCallback) -> *mut c_void;
 
+    pub fn create_error_callback(
+        error_callback: ErrorCallback,
+        userdata: *mut c_void,
+        userdata_deleter: ErrorUserdataDeleteCallback,
```

Comment:
Yeah. This would would be much simpler if we just leak everything.
It feels kinda bad to do that in a library like this though. Even if this allocation shouldn't matter much here. What if someone else uses the library in some other way?

Reply:
If we want to fix this, we should fix all of them in one go.

Reply:
so the problem here is the `userdata_deleter`? or is it the existence of the `create_error_callback` function?

Reply:
The deletion part. I think let's focus on the logging callback; and we can ensure all resources can be deleted in the next step.

Reply:
I'll have one more go at cleaning up the deletion part.
If that doesn't work I'll make a leaky version.
I just worry that it'll be hard to retrofit the deletion without introducing breaking changes to the api..